docs/relational-databases/extended-events/quick-start-extended-events-in-sql-server.md

```diff
@@ -49,7 +49,8 @@ After reading this article, you can:
 To get started, you need to:
 
 1. [Download SQL Server Management Studio (SSMS)](../../ssms/download-sql-server-management-studio-ssms.md). We recommend using a recent version of SSMS with the latest improvements and fixes.
-1. Ensure that your account has the `ALTER ANY EVENT SESSION` [server permission](../../t-sql/statements/grant-server-permissions-transact-sql.md).
+2. Ensure that your account has at least the following permisions: `CREATE ANY EVENT SESSION` or `ALTER ANY EVENT SESSION` [server permission](../../t-sql/statements/grant-server-permissions-transact-sql.md) on versions before SQL Sever 2022
```

Comment:
@AndreasWolter  This is unclear. Do you mean to suggest: In SQL Server 2022, `CREATE ANY EVENT SESSION`. Or before SQL Server 2022, `ALTER ANY EVENT SESSION`?

Reply:
I've made more clear and brought this commit internally.